skimage/io/_io.py

```diff
@@ -1,31 +1,34 @@
-from io import BytesIO
-
 import numpy as np
 import six
 
 from ..io.manage_plugins import call_plugin
 from ..color import rgb2grey
 from .util import file_or_url_context
 from ..exposure import is_low_contrast
-from .._shared.utils import all_warnings, warn
+from .._shared.utils import warn
 
 
 __all__ = ['imread', 'imsave', 'imshow', 'show',
            'imread_collection', 'imshow_collection']
 
 
-def imread(fname, as_grey=False, plugin=None, flatten=None,
+def imread(fname, as_gray=False, as_grey=None, plugin=None, flatten=None,
```

Comment:
Agreed, we should probably help out people like me who "misspell" grey frequently ;)